src/tasks/minions/woodcuttingActivity.ts

```diff
@@ -236,7 +236,7 @@ export const woodcuttingTask: MinionTask = {
 
 		let strungRabbitFoot = user.hasEquipped('Strung rabbit foot');
 		let twitchersEquipped = user.hasEquipped("twitcher's gloves");
-		let twitcherSetting = '';
+		let twitcherSetting: TwitcherGloves | undefined = undefined;
```

Comment:
I dont know if it should be or not, but you dont have the default here.

Reply:
I still have it as string in options. I fixed it. thanks for the catch

Reply:
oh i see what you're saying now, i just removed the default entirely so that no setting is selected. Do you think I should set this to a default? like `let twitcherSetting: TwitcherGloves | undefined = 'egg';`